src/Visualization/Python/PlotSlice.py

```diff
@@ -23,41 +23,46 @@
 
 
 def points_on_slice(
-    slice_origin: Sequence[float],
-    slice_radius: float,
-    slice_normal: Sequence[float],
-    slice_up: Sequence[float],
-    num_points: int,
+    slice_x: Sequence[float],
+    slice_y: Sequence[float],
+    num_points: Union[int, Sequence[int]],
+    slice_origin: Optional[Sequence[float]] = None,
+    slice_center: Optional[Sequence[float]] = None,
 ):
     """Returns points on a rectangular slice in 3D.
 
+    The slice is defined by either the origin or center, and two vectors in the
+    plane of the slice.
+
     Parameters:
-      slice_origin: Coordinates of the center of the slice.
-      slice_radius: Visible radius around the center of the slice.
-      slice_normal: Direction of the normal of the slice.
-      slice_up: Up-direction of the slice.
-      num_points: Number of points to return in each dimension.
+      slice_origin: Coordinates of the center of the slice. Mutually exclusive
+        with slice_center.
+      slice_center: Coordinates of the center of the slice. Mutually exclusive
+        with slice_origin.
+      slice_x: Horizontal axis of the slice. Must be orthogonal to slice_y.
+      slice_y: Vertical axis of the slice. Must be orthogonal to slice_x.
+      num_points: Number of points to return in each dimension (tuple or int).
```

Comment:
I actually think this is more confusing now than it was before. Even though it's explained in the CLI docs, I'm going to get confused every time what the difference between `slice_origin` and `slice_center` are. And I'm still a bit confused on what `slice_{x,y}` are. Also if you want to do anything besides a (1,0,0)/(0,1,0) slice, then it's going to be hard to ensure x/y are orthogonal.

I think the `slice_origin`, `slice_normal`, and `slice_up` are easier to understand and easier for users to input (in the case of a rotated xy-plane slice, you keep the normal the same and just change the "up" direction. With the new method, you have to calculate the vectors yourself before you run the CLI). What about replacing the name `slice_radius` with `slice_width` but keeping the functionality the same?

Reply:
ok, pushed a new fixup